components/filter/test/filter.test.js

```diff
@@ -68,7 +68,7 @@ describe('d2l-filter', () => {
 			await elem.updateComplete;
 
 			const infoMessage = elem.shadowRoot.querySelector('.d2l-filter-dimension-info-message');
-			expect(infoMessage.textContent).to.include('0 search results');
+			expect(infoMessage.textContent).to.include('No search results');
 			expect(infoMessage.classList.contains('d2l-offscreen')).to.be.false;
```

Comment:
@svanherk I think this is proof that the `zero` option was getting ignored in English, and now it works!

Reply:
Nice, surprised I never noticed that when I made them 🤦‍♀️